go-controller/pkg/cni/helper_linux.go

```diff
@@ -474,7 +474,7 @@ func (pr *PodRequest) deletePodConntrack() {
 				continue
 			}
 		}
-		err = util.DeleteConntrack(ip.Address.IP.String(), 0, "")
+		err = util.DeleteConntrack(ip.Address.IP.String(), 0, "", netlink.ConntrackReplyAnyIP)
```

Comment:
So deletePodConntrack is called from CNI cmd del. What happens if we have this kind of a scenario:
pod A on node 1
pod B on node 2

svc exists which is backed by pod B endpoint

pod A sends to svc this results in a conntrack entry on node A for DNAT svc to pod B. 
pod B is deleted, and removed as an endpoint. Now only the conntrack entry is flushed on on node B, even though the entry for DNAT to the endpoint is on node A?

If this is indeed an issue then we probably need to flush conntrack when we watch for endpoint deletion on every node. wdyt?

Reply:
you make a good point and I'd imagine that's a problem actually. It's possible the whole node which had the endpoint is gone but other nodes still have this entry on them...

Reply:
we can fix that in another PR?

Reply:
yes another PR along with that go routine stuff probably... since this is on endpoints deletion side...

Reply:
https://github.com/ovn-org/ovn-kubernetes/issues/2925